src/TorusUtilsExtraParams.ts

```diff
@@ -16,6 +12,13 @@ export interface TorusUtilsExtraParams {
   rpOrigin?: string; // passkey, webauthn
 
   rpId?: string; // passkey, webauthn
+}
+export interface TorusUtilsExtraParams extends TorusUtilsPasskeyExtraParams {
```

Comment:
Just question,
can we use just add TorusUtilsPasskeyExtraParams as a field in TorusUtilsExtraParams ?

```suggestion
export interface TorusUtilsExtraParams  {
     passkeyParams: TorusUtilsPasskeyExtraParams
```

Reply:
that's a good idea, we can do it in next breaking version. This change is not related to passkey linking. These are params for existing passkey/webauthn verifiers. I just divided it in seperate interface. We can do the change you are suggesting in next breaking version.